packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -188,9 +195,13 @@ const codeNativeIntegrationYaml = (
     { ...(configVars ?? {}) },
   );
 
-  const requiredConfigVars = Object.entries(configVarMap).map(([key, configVar]) =>
-    convertConfigVar(key, configVar, referenceKey, componentRegistry),
-  );
+  const requiredConfigVars: Array<ServerRequiredConfigVariable> = [];
+
+  Object.entries(configVarMap).forEach(([key, configVar]) => {
+    if ("dataType" in configVar && configVar.dataType !== "htmlElement") {
```

Comment:
I guess same question here. I'm not sure if maybe we can safely implicitly assume that is is or not, not being super familiar with this code myself.

Reply:
It should be but can add a check to be safe.